jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -74,6 +74,18 @@
             openstack-mkcloud-SLE12 (SLE12 host),
             openstack-mkcloud-SLE12-aarch64 (SLE12 aarch64 host)
 
+      - string:
```

Comment:
Could be done simpler:
create a variable called `debug` and offer these options:
```
- mkcloud
- qa_crowbarsetup
- all
```